skimage/filters/tests/test_fft_based.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import numpy as np
+from skimage.filters import butterworth
+
+
+def test_butterworth_2D():
+    im = np.zeros((4, 4))
+    filtered = butterworth(im)
+    assert filtered.shape == im.shape
+    assert np.all(im == filtered)
```

Comment:
Let me think about other possible tests for correctness.

The current tests are testing that things run without crashing for a range of dimensions / channel axes, but aren't really checking any property of the filtered image or comparing to a reference result.

Reply:
Yes I didn't have any good ideas on this. I thought first of constructing a dummy FFT by creating a zero array and then initializing a few pixels to 1, inverse fourrier transforming, then passing this image into the bw filter, then fourrier transforming it back. Unfortunately this no longer resembles the original FFT. I guess if one wisely constructs the FFT it should work?

Reply:
I came up with another basic test case that just verified highpass vs. lowpass behavior, but want to resolve the one remaining issue regarding `preserve_range` first.